spec/rubocop/cop/style/string_literals_spec.rb

```diff
@@ -194,6 +194,30 @@
       expect(cop.config_to_allow_offenses).to eq('Enabled' => false)
     end
 
+    it 'registers offense for escaped single quote in single quotes' do
+      inspect_source(%q('\''))
+      expect(cop.messages)
```

Comment:
Why aren't you simply using `expect_offense` here?

Reply:
Honestly, was just copy/pasting from the example above. Should I convert them to `expect_offense` as well while I'm on it?

Reply:
Yes, please. 🙂

Reply:
I have done this everywhere except the first example, because it checked more, and I wasn't sure how that works with `expect_offense`.
Pretty cool the way `expect_offense` works btw.